saml-core-api/pom.xml

```diff
@@ -64,6 +64,12 @@
         <dependency>
             <groupId>com.sun.xml.ws</groupId>
             <artifactId>rt</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.eclipse.angus</groupId>
+                    <artifactId>angus-activation</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
After the recent changes to this PR, do we still need this?

Reply:
AFAIK, some SAML tests failed previously, due to the conflict between the Jakarta Activation and Angus activation (probably). I'll check the results for this GHAs run.